src/components/premiumPricingCard.tsx

```diff
@@ -0,0 +1,77 @@
+import { Check } from "lucide-react";
+import { cn } from "~/lib/utils";
+
+import { Button } from "~/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "~/components/ui/card";
+
+const featureList = [
+  {
+    title: "Advanced Analytics",
+    description: "Get in-depth insights into your channel's performance.",
+  },
+  {
+    title: "Priority Support",
+    description: "Access premium customer support 24/7.",
+  },
+  {
+    title: "Unlimited upload speeds",
+    description: "Enjoy the fastest upload speeds possible.",
+  },
+];
+
+type CardProps = React.ComponentProps<typeof Card>;
+
+export default function CardStandard({ className, ...props }: CardProps) {
+  return (
+    <Card
+      className={cn(
+        "relative w-[360px] border-2 border-violet-500 shadow-lg shadow-fuchsia-900",
+        className,
+      )}
+      {...props}
+    >
+      <div className="absolute right-0 rounded-bl-md  bg-violet-500 p-1 font-semibold uppercase text-violet-100">
+        recommended
+      </div>
+      <CardHeader>
+        <CardTitle>
+          <span className="font-bold text-fuchsia-900">Premium</span> Plan
+        </CardTitle>
+        <CardDescription>
+          For large channels with even larger demands.
+        </CardDescription>
+      </CardHeader>
+      <CardContent className="mb-10 grid gap-4">
+        {featureList.map((feature, index) => (
+          <div
+            key={index}
```

Comment:
Overall `keys` on map should be unique and unchangable. Using and index is [not ok](https://react.dev/learn/rendering-lists#why-does-react-need-keys). In this scenario it's fine since it's static, but in scenarios if the order'd change it can lead to a unpredictable behavior. So it should be changed to a unique value between those objects. Like `title`, `description` or add a static value of `id`.

Same thing goes for `StandardPricingCard`